src/elements/content-sidebar/activity-feed/activity-feed/ActivityItem.scss

```diff
@@ -0,0 +1,7 @@
+@import '../../../common/variables';
+
+.bcs-ActivityItem {
+    .bcs-Comment {
+        padding: $sidebarActivityFeedSpacingVertical $sidebarActivityFeedSpacingHorizontal;
```

Comment:
minor: in the past Conrad mentioned that it was better to explicitly say `padding-left`, `padding-top`, etc., but it's up to you! Just a fyi

Reply:
Gotcha, not sure if we want to follow that in this case since it's 2 values for all 4 sides. Also we have variables saying `vertical` and `horizontal` makes it more readable. Being more explicit for 3 values+ makes sense.